packages/rancher-windows-exporter/charts/templates/configmap.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.clients }}{{ if .Values.clients.enabled }}
```

Comment:
Gated windows-exporter on `.Values.client.enabled` so disabling it on the rancher-monitoring level would allow you to disable deploying windows specific components